saleor/graphql/schema.graphql

```diff
@@ -41,7 +41,7 @@ input AddressInput {
   city: String
   cityArea: String
   postalCode: String
-  country: String!
+  country: String
```

Comment:
Is this change intentional?

Reply:
@maarcingebala  I am getting the same on master branch. It seems that some previous PR didnt include rebuilded schema file.

Reply:
It seems schema was not generated in #3891
The change is in the initial commit in the PR
https://github.com/mirumee/saleor/pull/3891/commits/31dc1eac3232ce0a86f79096b695bd8f122208b6#diff-556cb55b0af80084e456dbfff8ee36ccL26

Reply:
For reference this is the diff:
<img width="1397" alt="Screenshot 2019-04-11 at 14 14 06" src="https://user-images.githubusercontent.com/6186720/55956372-1b89a000-5c64-11e9-8c65-8b467d893d2f.png">

Does look accidental.

Reply:
I remember that I was reworking the address validation and maybe this change was some part of it. I guess country code doesn't need to be required at the schema level as address validator would check if it's set. Also, maybe it was easier to do the address updates without the country field being required, and that's was the reason for this change.

Reply:
@maarcingebala  So in that case, we can merge it. Am I right?

Reply:
I guess yes.

Reply:
Yep